packages/patrol/lib/src/native/native_automator2.dart

```diff
@@ -336,6 +336,16 @@ class NativeAutomator2 {
     );
   }
 
+  /// Press volume up
+  Future<void> pressVolumeUp() async {
+    await _wrapRequest('pressVolumeUp', _client.pressVolumeUp);
+  }
+
+  /// Press volume down
+  Future<void> pressVolumeDown() async {
+    await _wrapRequest('pressVolumeDown', _client.pressVolumeDown);
+  }
```

Comment:
Does those methods work on iOS Simulator and Android emulator? If no, please describe the limitation here

Reply:
It works on Android emulator, but doesn't run on iOS Simulator. I've added documentation for that